native-libs/deps/recipes/mymonerocorecpp/CMakeLists.txt

```diff
@@ -75,8 +75,8 @@ set(
     ${MONERO_SRC}/cryptonote_core/cryptonote_tx_utils.cpp
     ${MONERO_SRC}/common/base58.cpp
     ${MONERO_SRC}/common/aligned.c
-    # ${MONERO_SRC}/common/threadpool.cpp
-    # ${MONERO_SRC}/common/util.cpp
+    ${MONERO_SRC}/common/threadpool.cpp
```

Comment:
Are we running multiple threads now?

Reply:
Hi @andrejborstnik , the max concurrency thread is 1, so it seems not running as a multi-thread.